controller/protos/controller.proto

```diff
@@ -53,6 +53,20 @@ message DMXLight {
   Status status = 2;
 }
 
+// Configuration for badge reader.
+message Badger {
+  enum Status {
+    UNKNOWN = 0;
+    AUTHORIZED = 1;
+    UNAUTHORIZED = 2;
+  }
+  string url = 1;
+  string key_param = 2;
+  string usb_vendor_id = 3;
```

Comment:
add comment to say this should be in hex?

Reply:
Done.